package.json

```diff
@@ -6,12 +6,12 @@
   "repository": "http://github.com/ampproject/github-owners-bot",
   "description": "A web service that suggests approvers for a GitHub pull request based on OWNERS files and enforces reviews by OWNERS as a GitHub status check.",
   "engines": {
-    "node": "10.10.0",
-    "yarn": "^1.3.2"
+    "node": ">=8",
+    "yarn": ">=1.9"
```

Comment:
For consistency with `ampproject/amphtml`, this can be changed to:

```
  "engines": {
    "node": "^8.0.0 || ^10.0.0",
    "yarn": "^1.10.1"
  },
```